lib/vintage_net_wizard.ex

```diff
@@ -3,6 +3,20 @@ defmodule VintageNetWizard do
   Documentation for VintageNetWizard.
   """
 
+  @doc """
+  Run the wizard.
+
+  This means the WiFi module will be put into access point
+  mode and the web server will be started
+  """
+  @spec run_wizard() :: :ok
+  def run_wizard() do
+    with :ok <- into_ap_mode(),
```

Comment:
Did you want `run_wizard/0` to crash or return an error tuple? This line seems like error tuple and the next one is crash. I'm not quite sure which one I prefer, but I think I prefer one or the other.